docs/section19/19-14.md

```diff
@@ -0,0 +1,71 @@
+    - Create dictionary of hvac systems and associated zones and terminal units: `dict_of_zones_and_terminal_units_served_by_hvac_sys_b = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(B_RMI)`  
+    - Create dictionary with each zone's supply, return, exhaust, relief, terminal fan power as a list {zone.id: {"Fan_Power": [supply fan power kW, return fan power kW, exhaust fan power kW, relief fan power kW, terminal fan power]}}: `zone_supply_return_exhaust_relief_terminal_fan_power_dict = get_zone_supply_return_exhaust_relief_terminal_fan_power_dict(P_RMI)`  
+    - For each hvac system in the B_RMI: `for hvac in B_RMI...HeatingVentilatingAirConditioningSystem:` 
+        - Get the baseline system type: `sys_type = list(baseline_system_types_dict.keys())[list(baseline_system_types_dict.values()).index(hvac.id)])`
+        - Check if system type is of an applicable system type, if yes then carry on, if no then NOT_APPLICABLE for the hvac system:  `if any(baseline_system_type_compare(sys_type, target_sys_type, false) for target_system_type in target_system_types):`      
+            - Reset is modeled with a return fan in the proposed boolean variable: `is_modeled_with_return_fan_in_proposed = false`  
+            - Reset is modeled with a relief fan in the proposed boolean variable: `is_modeled_with_relief_fan_in_proposed = false`   
+            - Get list of zones served by the hvac system: `zones_list_hvac_sys_b = dict_of_zones_and_terminal_units_served_by_hvac_sys_b[hvac.id]["ZONE_LIST"]`  
+            - For each zone in the B_RMI check to see if the zone is served by an hvac system with return or relief fans in the proposed: `for zone in zones_list_hvac_sys_b:`  
+                - Get the analogous zone object from the proposed RMI (if this is not strictly the correct syntax please advise Weili): `zone_p = get_component_by_id(P_RMI,match_data_element(P_RMI,Zone,zone.id))`  
+                - Get list of the zone's supply, return, exhaust, and relief fan power modeled in the proposed: `modeled_fan_power_list_p = list(zone_supply_return_exhaust_relief_terminal_fan_power_dict[zone_p.id])`  
+                - Get the zone's total return fan kW, need to get the second element in the list of fan powers:`modeled_return_fan_power_p = modeled_fan_power_list_p[1]` 
+                - Get the zone's total relief fan kW, need to get the fourth element in the list of fan powers:`modeled_relief_fan_power_p = modeled_fan_power_list_p[3]` 
+                - Check if the total return fan power is greater than 0 in the proposed, if yes then set is_modeled_with_return_fan_in_proposed to true: `if modeled_return_fan_power_p > 0: is_modeled_with_return_fan_in_proposed = true`  
+                - Check if the total relief fan power is greater than 0 in the proposed, if yes then set is_modeled_with_relief_fan_in_proposed to true: `if modeled_relief_fan_power_p > 0: is_modeled_with_relief_fan_in_proposed = true`  
```

Comment:
I think we need to account for scenarios when, based on the above check, the baseline system must be modeled with both releif and return fans (i.e., some of the zones that it serves have relief fan in the proposed design, and some have return fan). I don't think the App G language is unclear about what to do in this case, so I'll start an exchange with the ECB. I see that you are interpreting it in Case 2 as requiring that both return and relief fans must be specified in the baseline. Perhaps this is reasonable.

Reply:
Discuss next step on call.